config.json

```diff
@@ -3,10 +3,12 @@
         "low": "1",
         "high": "8888"
     },
-
+    "iprange": {
```

Comment:
@Kashish121 - `ipRange` - follow the camel case for this file.

Reply:
Can you please clarify 'IPRange' or 'IpRange'?

Reply:
`IPRange` ❌ 

`IpRange` ❌ 

`ipRange` ✔️